gtfs/gtfs_utils/gtfs_utils/core_computations.py

```diff
@@ -23,6 +24,50 @@ def _read_almost_valid_csv_to_df(zip_path, file_name_in_zip, real_columns):
     return df
 
 
+def _fix_trip_to_date_columns(trip_date_df: pd.DataFrame) -> None:
+    """
+    Parse date and time columns
+    and fix rows where departure time is after midnight (hours after 24:00)
+    we only change the day of the week and the start time for those row, since the dates are correct
+    !! Does changes in place
+    """
+    DATE_FORMAT = '%d/%m/%Y 00:00:00'
+    trip_date_df['FromDate'] = pd.to_datetime(trip_date_df.FromDate, format=DATE_FORMAT)
+    trip_date_df['ToDate'] = pd.to_datetime(trip_date_df.ToDate, format=DATE_FORMAT)
+    trip_date_df['start_time'] = trip_date_df['start_time_str'].apply(parse_time_no_seconds_column)
+    series = trip_date_df['start_time'] >= 24 * 3600
+    trip_date_df.loc[series, 'DayInWeek'] = trip_date_df[series]['DayInWeek'] + 1
+    # fix dates to be from 0 o six (Sunday should be 1, Saturday 0)
+    trip_date_df['DayInWeek'] = trip_date_df['DayInWeek'] % 7
+    trip_date_df.loc[series, 'start_time'] = trip_date_df[series]['start_time'] % (24 * 3600)
+
+
+def get_trip_id_to_date_df(local_file_path: str, date: datetime.date) -> pd.DataFrame:
+    """
+    returns the TripIdToDate information that matches the given date
+    :param local_file_path: The path if the zip file
+    :param date: The date to filter by
+    :return: A DataFrame with the columns `route_id`, `trip_id_to_date` and `start_time`
+    """
+    trip_id_to_date_cols = ['route_id', 'OfficeLineId', 'Direction', 'LineAlternative', 'FromDate',
+                            'ToDate', 'trip_id_to_date', 'DayInWeek', 'start_time_str']
+    trip_date_df = _read_almost_valid_csv_to_df(local_file_path, TRIP_ID_TO_DATE_TXT_NAME, trip_id_to_date_cols)
+    _fix_trip_to_date_columns(trip_date_df)
+    # filter to a specific date
+    trip_date_df = trip_date_df[(trip_date_df['FromDate'] <= pd.Timestamp(date)) &
+                                (trip_date_df['ToDate'] >= pd.Timestamp(date)) &
+                                # Fix date.isoweekday to match israel counting (Sunday is 1, Saturday=0)
+                                (trip_date_df['DayInWeek'] == ((date.isoweekday() + 1) % 7))
+                                ]
+    # remove all columns except 'route_id' and 'trip_id_to_date'
+    trip_date_df = trip_date_df[['route_id', 'trip_id_to_date', 'start_time']]
```

Comment:
Get the output columns with a function parameter so it that it will be possible for someone to use this and get more columns. 
Leave these three columns as the default value.